tools/cli/app_test.go

```diff
@@ -410,10 +381,9 @@ func (s *cliAppSuite) TestRunWorkflow_Failed() {
 	resp := &types.StartWorkflowExecutionResponse{RunID: uuid.New()}
 	history := getWorkflowExecutionHistoryResponse
 	s.serverFrontendClient.EXPECT().StartWorkflowExecution(gomock.Any(), gomock.Any()).Return(resp, &types.BadRequestError{"faked error"})
-	s.serverFrontendClient.EXPECT().GetWorkflowExecutionHistory(gomock.Any(), gomock.Any()).Return(history, nil)
+	s.serverFrontendClient.EXPECT().GetWorkflowExecutionHistory(gomock.Any(), gomock.Any()).Return(history, nil).Times(0)
```

Comment:
deleting this probably works too, but IMO fine if you prefer it as documentation that this specifically is not called.